nap/utils.py

```diff
@@ -46,12 +32,20 @@ def make_url(base_url, params=None, add_slash=None):
 
     if params:
 
+        def safe_encode(value):
+            try:
+                if isinstance(value, unicode):
```

Comment:
I had this in a try/except because the encoding can fail for some data types. But Since I'm now only encoding unicode strings, This probably doesn't need to be wrapped in a try anymore.